ethicalads-theme/templates/ea/includes/advertiser-testimonials.html

```diff
@@ -5,7 +5,7 @@
   <div class="row">
     <div class="col mb-3">
       <h2 class="text-center font-weight-bold mb-8">Over 1 billion ads served since 2016</h2>
-      <p class="text-center lead"><strong>Advertisers</strong> have success by targeting developers on the sites they frequent, without being blocked.</p>
+      <p class="text-center lead">We help <strong>tech marketers</strong> succeed in reaching developers on the sites they frequent, without being blocked.</p>
```

Comment:
```suggestion
      <p class="text-center lead">We help <strong>tech marketers</strong> reach developers on the sites they frequent, without being blocked.</p>
```

Feels more readable?

Reply:
Agreed.